8.x/getting-started/contributing.md

```diff
@@ -385,6 +385,115 @@ ___
 
 **10.** For every document that is written and placed in the documentation section of the Larammerce site, the members of the Larammerce team are obliged to read it.
 
+## PR review
+
+This section describes how to run, test and review your teammate's pull request in your local system.
+
+### Git
+
+`Git`*<sup>[4](#4)</sup>* is a `version control` or `source control` system which saves the source code and remembers the source code `changes` step by step as a stack. This stack is called `branch` and the changes are called `commit` or `revision`.
+
+If you order to cancel a commit, git creates a new commit which reverts the previous commit changes. So no commit is deleted from the git, but always the new commits are added. Git performs this task by adding and subtracting the lines. For example `+5 -3` in a given commit means that you added 5 lines to the source code and at the same time deleted 3 lines from it. Now if you order to delete this commit, git deletes the 5 new lines and adds the previous 3 lines and creates a new commit.
+
+In the git platform, the first branch is called `master` or `main`. A new lateral branch can be created and initialized from any given commit in the stack and, after several steps, merged into the master branch as a new commit.
+
+##### EXAMPLE
+
+Suppose you write the codes below:
+
+```bash
+#include<iostream>
+using namespace std;
+int main(){
+    cout<< "Hello!";
+    return 0;
+}
+```
+
+When you commit these codes, git makes a revision number 0 and assigns it a random string, like 223465:
+
+223465 0 -> Create file main.cpp -> +6 0
+
+`+6 0` means that you added 6 new lines and no line is deleted.
+
+Now you change the code as follows and commit it:
+
+```bash
+#include<iostream>
+int main(){
+    std::cout<< "Hello!";
+    return 0;
+}
+```
+
+Git makes the revision number 1 with a new random string, such as 67fe45:
+
+67fe45 1 -> Do some changes. -> +1 -2
+
+`-2` means that 2 lines are deleted: `using namespace std;` and `cout<< "Hello!";`.
+
+`+1` means that 1 new line is added: `std::cout<< "Hello!";`.
+
+Now you order to revert the last revision:
+
+```bash
+git revert 67fe45
+```
+
+Git does not delete the revision 67fe45, but makes the revision number 2 with a new random string, like 879012:
```

Comment:
Instead of deleting the revision 67fe45, Git generates a new random string, such as 879012, and assigns it to a new revision number, namely 2: